packages/entities/indexedDb.ts

```diff
@@ -4,7 +4,7 @@ export interface IConfig {
 }
 
 export interface IVotes {
```

Comment:
doesn't this represent a single vote?
so i guess  
```suggestion
export interface IVote {
```

Reply:
done